test-data/unit/lib-stub/attr.pyi

```diff
@@ -1,57 +1,98 @@
-from typing import TypeVar, overload, Callable, Any, Type, Optional
+from typing import TypeVar, overload, Callable, Any, Type, Optional, Union, Sequence, Mapping
 
 _T = TypeVar('_T')
 _C = TypeVar('_C', bound=type)
 
+_ValidatorType = Callable[[Any, Any, _T], Any]
```

Comment:
I thought validator should return `bool`, isn't this the case?

Reply:
The return value of validators is ignored.  A validation error is a raised exception.   However I don't want to force a "None"